WooCommerce/WooCommerceTests/ViewRelated/Products/Media/ProductImageUploaderTests.swift

```diff
@@ -135,4 +135,37 @@ final class ProductImageUploaderTests: XCTestCase {
         let images = try XCTUnwrap(resultOfSavedImages.get())
         XCTAssertEqual(images.map { $0.imageID }, [606, 645])
     }
+
+    func test_replaceLocalID_replaces_productID_properly() {
```

Comment:
nice test! how about another test case to make sure `replaceLocalID` with a non-existing key does nothing / doesn't crash the app?

Reply:
Done in f8ee3f170c50f8e6e6b240d31671d63d2b9c932f